examples/gocean/eg5/README.md

```diff
@@ -65,6 +72,10 @@ flag, which will add profiling around the two invokes used in the example.
 The actual runtime is extremely short, so likely the profiling
 tool used will report 0 seconds for each of the invokes.
 
+### Note for LFRic wrapper library
+The LFRic timer library writes its output to a file ``timer.txt``
+(and it will overwrite this file if it should already exist).
+
 ## Running
 The output will depend on the wrapper library used. For the ``template``
 library, you should see:
```

Comment:
Please could you change the copyright (below) to 2020-2021.

Reply:
Done.